opentelemetry-sdk/src/opentelemetry/sdk/_logs/_internal/__init__.py

```diff
@@ -169,6 +169,8 @@ def __init__(
         attributes: Optional[Attributes] = None,
         limits: Optional[LogLimits] = _UnsetLogLimits,
     ):
+        if not observed_timestamp:
+            observed_timestamp = time_ns()
```

Comment:
Does this change need to happen only in _both_ API and SDK?

Reply:
I think it is fine that it only happens in the API. The spec document where this is defined is `bridge-api.md`. This is not infallible, I remember spec requirements defined in API/SDK files that had to be implemented in the SDK/API components respectively, but for this situation I would expect this to be implemented only in the API.